framework/src/problems/DumpObjectsProblem.C

```diff
@@ -143,6 +179,18 @@ DumpObjectsProblem::dumpVariableHelper(const std::string & system,
 
 void
 DumpObjectsProblem::solve(unsigned int)
+{
+  printObjects();
+}
+
+void
+DumpObjectsProblem::solveLinearSystem(unsigned int, const Moose::PetscSupport::PetscOptions *)
+{
+  printObjects();
```

Comment:
Do we need these calls anymore if the action is doing the `printObjects` call?

Reply:
A derived class (not sure anyone had one) could be using the old solve to print

Reply:
Yea I'm confident we don't have anyone deriving from `DumpObjectsProblem`

Reply:
we still need to override solve imo to avoid trying to solve without initialSetup called. but should be fine now